internal/backend.go

```diff
@@ -315,6 +316,7 @@ func initializeSensors(controllers []*hwmon.HwMonController) error {
 				return fmt.Errorf("couldn't find hwmon device with platform '%s' for sensor: %s. Run 'fan2go detect' again and correct any mistake", config.HwMon.Platform, config.ID)
 			}
 		}
+		// TODO: nvidia?
```

Comment:
by the way, what happens above in the HwMon case if the "platform" was found, but it has no sensor at `config.HwMon.Index`?

Reply:
I think I fixed that case (and the equivalent ones for the nvidia backend)